scripts/publish-msix.ps1

```
@@ -0,0 +1,97 @@
+  # One-time dev cert + manifest alignment, then publish:
+  .\scripts\new-msix-dev-cert.ps1 -UpdateManifests
+  .\scripts\publish-msix.ps1 -CertificatePath "Desktop\HermesDesktop\packaging\dev-msix.pfx" -CertificatePassword dev
+
+.EXAMPLE
+  .\scripts\publish-msix.ps1 -CertificatePath "$env:USERPROFILE\certs\HermesDesktop.pfx"
+#>
+param(
+    [Parameter(Mandatory = $true)]
+    [string] $CertificatePath,
+
+    $CertificatePassword,
```

PSScriptAnalyzer (1.25.0)

[warning] Parameter '$CertificatePassword' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $CertificatePassword type to SecureString; Set $CertificatePassword type to PSCredential

(PSAvoidUsingPlainTextForPassword)